tests/math/test_matrix_manipulation.py

```diff
@@ -197,7 +207,7 @@ def test_autograd(self, i, base_matrix, tol):
     def test_torch(self, i, base_matrix, tol):
         """Tests differentiation in torch by computing the Jacobian of
         the expanded matrix with respect to the canonical matrix."""
-        import torch
+        import torch  # pylint: disable=Import outside toplevel (torch) (import-outside-toplevel)
```

Comment:
```suggestion
        import torch
```

The pylint exception is not be necessary as it is already in `tests/.pylintrc` file (see [here](https://github.com/PennyLaneAI/pennylane/blob/master/tests/.pylintrc#L43). To convince yourself, try running 
```
pylint --rcfile tests/.pylintrc tests/math/test_matrix_manipulation.py
```
locally or push without these comments and see how CI does not raise failures :)

Please also remove them from the other additions below and leave the other tests as is